.bazelrc

```diff
@@ -25,6 +26,9 @@ build --define=with_xla_support=false
 # Disable MKL
 build --define=enable_mkl=false --define=build_with_mkl=false --define=tensorflow_mkldnn_contraction_kernel=0
 
+# Disable XNNPACK on elinux_armhf due to https://github.com/tensorflow/tensorflow/issues/57364
+build:elinux_armhf --define tflite_with_xnnpack=false
```

Comment:
As far as I understand, this is for 32-bit ARM, right? So should a similar define then also be added to the 32-bit RPi3 configuration below?

Reply:
The 32-bit RPi3 config inherits from `elinux_armhf` so this will automatically be applied there as well:
https://github.com/larq/compute-engine/blob/84e30bd80e18d2aeba923d6c44d40098b8702184/.bazelrc#L29